parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -182,7 +182,7 @@ ps auxfww > /opt/azure/provision-ps.log &
 
 if $FULL_INSTALL_REQUIRED; then
   if [[ $OS == $UBUNTU_OS_NAME ]]; then
-    applyCIS || exit $ERR_CIS_HARDENING_ERROR
```

Comment:
this was never actually getting called. The function themselves didn't have any `return 1`  statements.